### ISSUE TITLE
Deprecate rename and count

### ISSUE THREAD

**Issue by krlmlr** [hadley/plyr#252]
for full (!) forward compatibility with `dplyr`. See http://rpubs.com/krlmlr/pdlyr for an analysis.

**Comment by hadley** [comment]
I don't want to do that in this release as it would require a lot of downstream changes, but it seems like a reasonable idea.

**Comment by krlmlr** [comment]
Can we do it right after CRAN release then?

**Comment by hadley** [comment]
Yeah, although we'll need a plan for informing downstream maintainers. It would be helpful if you could estimate the amount of work by deprecating those functions and then running the revdep checks

**Comment by krlmlr** [comment]
How about adding now compatibility symbols in this release (`rename` -> `prename` et al.), to faciliate a migration?

I'll do the revdep checks.

**Comment by hadley** [comment]
Hmmmm, anything that requires deprecating existing (and popular) plyr functions is going to cause a lot of pain. I wonder if there's another approach

**Comment by krlmlr** [comment]
I guess you don't want to rename the corresponding `dplyr` verbs either. We could as well keep everything as is, and explicitly state this incompatibility. Do you think it's worth pursuing the compatibility package I started? I think such a separate glue is preferable to including this into `dplyr`.

By the same token -- there seem to be many requests for convenience functions in `dplyr` that can be implemented using the API. Do we already have a `dplyrAlly`?

**Comment by krlmlr** [comment]
`revdep_check` is flaky on my system, I'm trying for the third time now. There's https://github.com/skranz/dplyrExtras.
